app/views/projects/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<h2>Project Name: <%= @project.name %></h2>
+
+<%= form_with url: "/projects/#{@project.id}", method: :get, local: true, data: {turbo:false} do |f| %>
```

Comment:
No need to include `local: true` if you have disabled Turbo

Reply:
this is where you could refactor your routes to make them more RESTful. This form could go to 
"/projects/:project_id/contestants" and could be a POST, which could be routed to a `create` action in your joins controller (ContestantProjectsController)